File: LV.1/K번째 수/solution.py
def solution(array, commands):
		last_arr = []

		for i in commands:                              
		    new_arr = array[i[0]-1:i[1]]       
		    new_arr.sort()                     
                                   
		    last_arr.append(new_arr[i[2]-1])   

		return last_arr

File: LV.1/K번째 수/Solution.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        int[] new_arr = {};
        int[] temp_arr = {};
        
        for (int i=0; i<commands.length; i++) {
            temp_arr = commands[i];
            
            new_arr = Arrays.copyOfRange(array,temp_arr[0]-1,temp_arr[1]);
            Arrays.sort(new_arr);
            
            answer[i] = new_arr[temp_arr[2]-1];
        }
        
        return answer;
    }
}
